lib/job.jsonnet

```diff
@@ -302,6 +306,21 @@ function(jobName, agentEnv={}, stepEnvFile='', patchFunc=identity) patchFunc({
         {name: env.BUILDKITE_PLUGIN_K8S_IMAGE_PULL_SECRET},
     ],
 
+  local fsGroup =
+    if env.BUILDKITE_PLUGIN_K8S_BUILD_DIRECTORY_FSGROUP == '' then {}
+    else 
+      {fsGroup: std.parseInt(env.BUILDKITE_PLUGIN_K8S_BUILD_DIRECTORY_FSGROUP)},
+  
```

Comment:
notes to self: this needs to be removed since it never gets exposed via the plugin